app/assets/javascripts/application.js

```diff
@@ -41,4 +41,38 @@ function show_notification_blood(msg){
   });
 }
 
+ function initialize() {
+  var input = document.getElementById('searchTextField');
 
+  var options = {
+    componentRestrictions: {country: "eg"}
+  };
+
+  var autocomplete = new google.maps.places.Autocomplete(input,options);
+  google.maps.event.addListener(autocomplete, 'place_changed', function () {
+    var place = autocomplete.getPlace();
+        document.getElementById('addName').value = document.getElementById('searchTextField').value;
+        document.getElementById('addLat').value = place.geometry.location.lat();
+        document.getElementById('addLng').value = place.geometry.location.lng();
+   });
+}
+
+function initializeHospital() {
+
+  var input = document.getElementById('hospital_searchTextField');
+  var options = { 
+      componentRestrictions: {country: "eg"}
+   };
+
+  var autocomplete = new google.maps.places.Autocomplete(input , options);
+  google.maps.event.addListener(autocomplete, 'place_changed', function () {
+    var place = autocomplete.getPlace();
+    document.getElementById('hospital_addName').value = document.getElementById('hospital_searchTextField').value;
+    document.getElementById('hospital_addLat').value = place.geometry.location.lat();
+    document.getElementById('hospital_addLng').value = place.geometry.location.lng();
+  });
+}
+
+ $(function() {
+          $( ".dtpicker" ).datepicker({changeYear: true, yearRange : '-100:+0',dateFormat: 'yy-mm-dd'});
+        });
```

Comment:
Indentation 

$(function() {
          $( ".dtpicker" ).datepicker({changeYear: true, yearRange : '-100:+0',dateFormat: 'yy-mm-dd'});
         });

Better to be

$(function() {
   $( ".dtpicker" ).datepicker({changeYear: true, yearRange : '-100:+0',dateFormat: 'yy-mm-dd'});
});